src/overlays/actors/ovl_Bg_Spot03_Taki/z_bg_spot03_taki.c

```diff
@@ -27,15 +31,141 @@ const ActorInit Bg_Spot03_Taki_InitVars = {
     (ActorFunc)BgSpot03Taki_Update,
     (ActorFunc)BgSpot03Taki_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot03_Taki/func_808ADAE0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot03_Taki/BgSpot03Taki_Init.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+};
+
+extern UNK_TYPE D_06000C98;
+extern Vtx* D_06000800[];
+extern Vtx* D_06000990[];
+extern Gfx* D_06000B20[];
+extern Gfx* D_06001580[];
+
+void func_808ADAE0(BgSpot03Taki* this, s32 arg0) {
```

Comment:
Thinking of calling this function something like BgSpot03Taki_ApplyOpeningAlpha since it's not _setting_ the alpha, just applying it to the current vertex buffer. Or is it not kosher to be renaming functions right now?